cmake/px4_add_board.cmake

```diff
@@ -262,6 +266,12 @@ function(px4_add_board)
 	if(CONSTRAINED_FLASH)
 		set(px4_constrained_flash_build "1" CACHE INTERNAL "constrained flash build" FORCE)
 		add_definitions(-DCONSTRAINED_FLASH)
+		if (NO_HELP)
+			if (NOT HELP_URL)
+				set(HELP_URL https://docs.px4.io/master/)
```

Comment:
Also set to https://docs.px4.io/master/en/

Reply:
"